server/mtdnetwork/component/host.py

```
@@ -675,7 +681,7 @@ def get_path_from_exposed(self):
                 if path_len < shortest_distance:
                     shortest_distance = path_len
                     shortest_path = path
-            except:
+            except Exception:
                 pass
```

can you  please throw an error or log that an error has occured here.